react/next/SliderTrack.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { FC } from 'react'
+import { useSpring, animated, config as springPresets } from 'react-spring'
+
+interface Props {
+  className?: string
+  transform: number
+  shouldRenderOnSSR: boolean
+}
+
+const SliderTrack: FC<Props> = ({
+  children,
+  className,
+  transform,
+  shouldRenderOnSSR,
+}) => {
+  const animation =
+    !!window.requestAnimationFrame &&
+    useSpring({
+      config: springPresets.default,
+      transform: `translate3d(${transform}px, 0, 0)`,
+    })
+
+  return (
+    <animated.div
```

Comment:
can you use css transitions here, and avoid importing `react-spring`?

Reply:
Done